src/main/antlr4/com/github/palindromicity/syslog/dsl/generated/Rfc5424.g4

```diff
@@ -37,9 +37,7 @@ grammar Rfc5424;
 
 }
 
```

Comment:
yes, this looks right.
Extra credit if you want to make a new rule:

octet_prefix : nonzero_digit digit*;

and then change octet_prefixed and heroku_https_log_drain to start with it :)

Reply:
If not I'll do it as a follow on before release

Reply:
maybe the name should be 6587_prefix

Reply:
Antlr didn't like the grammar starting with a number, we could do rfc6587_prefix ?

Reply:
yup!